app/assets/stylesheets/components/firm/_all.scss

```diff
@@ -1,2 +1,3 @@
 @import 'settings';
 @import 'firm';
+@import 'firm_list';
```

Comment:
Really `firm_list` should be moved into its own directory within components, and have its own settings file.